controllers/volumereplicationgroup_controller_test.go

```diff
@@ -50,11 +55,19 @@ var _ = Describe("Test VolumeReplicationGroup", func() {
 			}
 		})
 		It("waits for VRG to create a VR for each PVC", func() {
+			totalPVCCount := 0
 			for c := 0; c < len(vrgTestCases); c++ {
 				v := vrgTestCases[c]
 				expectedVRCount := len(v.pvcNames)
 				v.waitForVRCountToMatch(expectedVRCount)
+
+				totalPVCCount += expectedVRCount
```

Comment:
Here does totalPVCCount mean all the PVCs from all the VRGs? Or just PVCs protected by an individual VRG? I assume the former.

Reply:
`totalPVCCount` counts all the PVCs across all the VRGs used in that test